src/pyenphase/models/inverter.py

```diff
@@ -0,0 +1,29 @@
+"""Model for an Enphase microinverter."""
+
+from typing import Any
+
+
+class EnvoyInverter:
```

Comment:
There will like be a lot of these.

We could use `__slots__` to save memory

Reply:
Never actually used slots before, is it just as simple as what I did in the latest commit? 🙃